hyperlane/relayer.go

```diff
@@ -42,48 +42,48 @@ var (
 	ErrNotEnoughSignaturesFound = errors.New("number of signatures found in multisig signed checkpoint is below expected threshold")
 )
 
-func (r *relayer) Relay(ctx context.Context, originChainID string, initiateTxHash string, maxTxFeeUUSDC *big.Int) (destinationTxHash string, destinationChainID string, err error) {
+func (r *relayer) Relay(ctx context.Context, originChainID string, initiateTxHash string, maxTxFeeUUSDC *big.Int) (destinationTxHash string, destinationChainID string, rawTx string, err error) {
```

Comment:
Not necessary and more of a personal stylistic thing, but imo when we are returning a bunch of the same type from a func and use named returns just to identify the return types, it would probably be better to return a struct that wrap those types, so they can have names that way, so something like 
```go
type RelayedTransaction struct {
    DestinationTxHash string
    DestinationChainID string
    RawTxBase64 string
    // ... possibly origin info too
}
Relay(ctx context.Context, originChainID string, initiateTxHash string, maxTxFeeUUSDC *big.Int) (*RelayedTransaction, error)
```

Reply:
Yeah we probably have different stylistic opinions here. Imo moving it to a struct doesn't meaningfully improve readability while also adding an extra level of indirection for people trying to grok the interface.